static/js/components/widgets/ResourcePickerListing.test.tsx

```diff
@@ -156,6 +174,28 @@ describe("ResourcePickerListing", () => {
     ).toBe("/path/to/image.jpg")
   })
 
+  it("should fetch and display pages", async () => {
+    const { wrapper } = await render({
+      contentType:  "page",
+      resourcetype: null
+    })
+    expect(
+      wrapper
+        .find(".resource-picker-listing .resource-item")
+        .map(el => el.find("h4").text())
+    ).toEqual(contentListingItems[1].map(item => item.title))
```

Comment:
here it might be good to add an assertion that the API request we expected went through, because I think looking at the API mocks up top we're returning `contentListingItems[1]` for both document and page requests. An alternative I guess would be to add a new, separate return value just for the pages, either way would disambiguate it I think